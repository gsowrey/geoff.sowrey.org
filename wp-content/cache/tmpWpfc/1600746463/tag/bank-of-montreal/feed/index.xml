<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	
	xmlns:georss="http://www.georss.org/georss"
	xmlns:geo="http://www.w3.org/2003/01/geo/wgs84_pos#"
	>

<channel>
	<title>bank of montreal &#8211; The Observer&#039;s Log</title>
	<atom:link href="https://geoff.sowrey.org/tag/bank-of-montreal/feed/" rel="self" type="application/rss+xml" />
	<link>https://geoff.sowrey.org</link>
	<description>A miscellany of know-it-all-isms by Geoff Sowrey</description>
	<lastBuildDate>Mon, 31 Dec 2007 14:58:43 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.4</generator>
<site xmlns="com-wordpress:feed-additions:1">157010562</site>	<item>
		<title>BMO locked me out AGAIN</title>
		<link>https://geoff.sowrey.org/2007/12/bmo-locked-me-out-again/?utm_source=rss&#038;utm_medium=rss&#038;utm_campaign=bmo-locked-me-out-again</link>
					<comments>https://geoff.sowrey.org/2007/12/bmo-locked-me-out-again/#comments</comments>
		
		<dc:creator><![CDATA[Geoff]]></dc:creator>
		<pubDate>Mon, 31 Dec 2007 14:58:43 +0000</pubDate>
				<category><![CDATA[General]]></category>
		<category><![CDATA[Rants]]></category>
		<category><![CDATA[bank of montreal]]></category>
		<category><![CDATA[banks]]></category>
		<category><![CDATA[internet]]></category>
		<category><![CDATA[security]]></category>
		<guid isPermaLink="false">http://www.sowrey.org/2007/12/31/bmo-locked-me-out-again/</guid>

					<description><![CDATA[GAH! [[Bank of Montreal, I hate your security&#124;I already hate BMO&#8217;s online security.]] Now I hate it more, if it is in fact possible to hate something more than &#8220;totally&#8221;. I just wanted to into my account. Nothing serious. But the password they made me assign apparently has changed, or is something completely different than &#8230; <a href="https://geoff.sowrey.org/2007/12/bmo-locked-me-out-again/" class="more-link">Continue reading<span class="screen-reader-text"> "BMO locked me out AGAIN"</span></a>]]></description>
										<content:encoded><![CDATA[<p>GAH!<br />
[[Bank of Montreal, I hate your security|I already hate BMO&#8217;s online security.]] Now I hate it more, if it is in fact possible to hate something more than &#8220;totally&#8221;.<br />
I just wanted to into my account. Nothing serious. But the password they made me assign apparently has changed, or is something completely different than I can remember. It&#8217;s not like I already have 8,349,204 passwords that I need to remember (thank you, Sarbanes-Oxley) and I must&#8217;ve made something up on the spot, wrote it down and then immediately burned the paper without looking at what I wrote.<br />
That must&#8217;ve been it. No way it could be BMO. Not a chance.<br />
Augh.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://geoff.sowrey.org/2007/12/bmo-locked-me-out-again/feed/</wfw:commentRss>
			<slash:comments>9</slash:comments>
		
		
		<post-id xmlns="com-wordpress:feed-additions:1">882</post-id>	</item>
		<item>
		<title>Bank of Montreal, I hate your security</title>
		<link>https://geoff.sowrey.org/2007/11/bank-of-montreal-i-hate-your-security/?utm_source=rss&#038;utm_medium=rss&#038;utm_campaign=bank-of-montreal-i-hate-your-security</link>
					<comments>https://geoff.sowrey.org/2007/11/bank-of-montreal-i-hate-your-security/#comments</comments>
		
		<dc:creator><![CDATA[Geoff]]></dc:creator>
		<pubDate>Wed, 14 Nov 2007 19:41:57 +0000</pubDate>
				<category><![CDATA[General]]></category>
		<category><![CDATA[Rants]]></category>
		<category><![CDATA[bank of montreal]]></category>
		<category><![CDATA[banks]]></category>
		<category><![CDATA[internet]]></category>
		<category><![CDATA[security]]></category>
		<guid isPermaLink="false">http://www.sowrey.org/2007/11/14/bank-of-montreal-i-hate-your-security/</guid>

					<description><![CDATA[Oh. My. Lord. I haven&#8217;t been this pissed off in a very, very long time. I actually yelled at the poor person on the phone trying to help me. It&#8217;s not her fault. It&#8217;s the Bank of Montreal. Who obviously has NO FUCKING CLUE what it&#8217;s like to live in the digital age. Allow me &#8230; <a href="https://geoff.sowrey.org/2007/11/bank-of-montreal-i-hate-your-security/" class="more-link">Continue reading<span class="screen-reader-text"> "Bank of Montreal, I hate your security"</span></a>]]></description>
										<content:encoded><![CDATA[<p>Oh. My. Lord.<br />
I haven&#8217;t been this pissed off in a very, very long time. I actually yelled at the poor person on the phone trying to help me. It&#8217;s not her fault. It&#8217;s the <a href="http://www.bmo.com">Bank of Montreal</a>. Who obviously has <strong>NO FUCKING CLUE</strong> what it&#8217;s like to live in the digital age.<br />
Allow me to elaborate&#8230;<br />
<span id="more-860"></span>BMO (the short form of Bank of Montreal) recently introduced new security procedures for their online banking. In addition to your card number and PIN, you choose an image and provide a password. (This is all to prevent phishing.) I welcomed this, since I had similar functionality on my <a href="http://www.ingdirect.ca/">ING</a> account for a couple of years.<br />
So, choose an image, provide a little info, toss in a password. No problem.<br />
Until I came back. Suddenly, my previously wonderful password doesn&#8217;t work. No idea why. Could swear I&#8217;ve got the right one. Try again. And again. And suddenly, I get a page telling me that my card is locked (thankfully, only online) and to call BMO to have it unlocked.<br />
Which I got around to today. And boy, I can&#8217;t think of a worse experience. Ever.<br />
When you need access to personal financial information through your institution, you generally get asked the Big 6:</p>
<ul>
<li>Name</li>
<li>Birthday</li>
<li>Address</li>
<li>Phone number</li>
<li>Social Insurance (or Social Security) Number</li>
<li>Mother&#8217;s maiden name</li>
</ul>
<p>Now, admittedly, anyone wanting to steal your identity could dig this information up and put it to nefarious means. So to that extent, I see that BMO&#8217;s approach for more information is necessary.<br />
Their additional questions:</p>
<ul>
<li>What investments do you have? (I have none with BMO, so this question is invalid)</li>
<li>What are your monthly mortgage payments?</li>
<li>What were the last three transactions in your account &#8212; including the name of the institution and the dollar amount?</li>
</ul>
<p>I need to get at least two of those right in order to be validated as me. Problem: <em>I DON&#8217;T HAVE ACCESS TO MY FUCKING ACCOUNT!</em>  So how the hell am I supposed to know?<br />
No, I don&#8217;t memorise <strong>every bloody number</strong> that goes in and out of my account. That&#8217;s what <em>monthly statements</em> are for. Or, better yet, the online banking website that <em>I don&#8217;t have access to!</em> I don&#8217;t want to know what my monthly mortgage payments are &#8212; I just want to know that they&#8217;re being paid, and that my mortgage is being paid down.<br />
This is the digital age, folks. People actively try not to know numbers. I can&#8217;t count the number of people I know who don&#8217;t know their own <em>phone numbers</em>, let alone their last three transactions. Besides, as you already know (because you asked) &#8212; <em>I&#8217;m married</em>, so <em>I don&#8217;t know the last three transactions because I might not have made them!</em><br />
This is an open letter to BMO: Find a process that does not require someone to have up-to-the-moment financial history. It&#8217;s not feasible, and it&#8217;s (obviously) too difficult for some of us poor schmucks who are more worried about forgetting their wife&#8217;s birthday.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://geoff.sowrey.org/2007/11/bank-of-montreal-i-hate-your-security/feed/</wfw:commentRss>
			<slash:comments>24</slash:comments>
		
		
		<post-id xmlns="com-wordpress:feed-additions:1">860</post-id>	</item>
	</channel>
</rss>
