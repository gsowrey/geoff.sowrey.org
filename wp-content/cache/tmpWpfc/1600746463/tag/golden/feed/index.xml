<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	
	xmlns:georss="http://www.georss.org/georss"
	xmlns:geo="http://www.w3.org/2003/01/geo/wgs84_pos#"
	>

<channel>
	<title>golden &#8211; The Observer&#039;s Log</title>
	<atom:link href="https://geoff.sowrey.org/tag/golden/feed/" rel="self" type="application/rss+xml" />
	<link>https://geoff.sowrey.org</link>
	<description>A miscellany of know-it-all-isms by Geoff Sowrey</description>
	<lastBuildDate>Mon, 17 Jul 2006 16:03:43 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.4</generator>
<site xmlns="com-wordpress:feed-additions:1">157010562</site>	<item>
		<title>Surprise birthday weekend in Revelstoke</title>
		<link>https://geoff.sowrey.org/2006/07/surprise-birthday-weekend-in-revelstoke/?utm_source=rss&#038;utm_medium=rss&#038;utm_campaign=surprise-birthday-weekend-in-revelstoke</link>
					<comments>https://geoff.sowrey.org/2006/07/surprise-birthday-weekend-in-revelstoke/#respond</comments>
		
		<dc:creator><![CDATA[Geoff]]></dc:creator>
		<pubDate>Mon, 17 Jul 2006 16:03:43 +0000</pubDate>
				<category><![CDATA[General]]></category>
		<category><![CDATA[Trains]]></category>
		<category><![CDATA[Travel]]></category>
		<category><![CDATA[banff]]></category>
		<category><![CDATA[birthday]]></category>
		<category><![CDATA[golden]]></category>
		<category><![CDATA[revelstoke]]></category>
		<category><![CDATA[surprise]]></category>
		<category><![CDATA[three valley gap]]></category>
		<guid isPermaLink="false">http://www.sowrey.org/index.php/2006/07/17/surprise-birthday-weekend-in-revelstoke/</guid>

					<description><![CDATA[This weekend, it was my turn to be surprised. And surprises came in droves. Sunday was my 34th birthday. I find it a little hard to believe that I&#8217;m actually that old. I always figured when I was in my 30s, I&#8217;d be more &#8230; mature. And yet I find myself being more immature with &#8230; <a href="https://geoff.sowrey.org/2006/07/surprise-birthday-weekend-in-revelstoke/" class="more-link">Continue reading<span class="screen-reader-text"> "Surprise birthday weekend in Revelstoke"</span></a>]]></description>
										<content:encoded><![CDATA[<p>This weekend, it was my turn to be surprised. And surprises came in droves.<br />
Sunday was my 34th birthday. I find it a little hard to believe that I&#8217;m actually that old. I always figured when I was in my 30s, I&#8217;d be more &#8230; mature. And yet I find myself being more immature with each passing year. I guess I&#8217;m holding to my pattern &#8212; 18 years old, with a heap of experience. For now, I think that&#8217;s a good thing. Experience helps, and one can&#8217;t get too mature. You lose all the fun, after all!<br />
<span id="more-441"></span>I sprinted home from work. I can bike home now almost as fast as I used to drive. I can&#8217;t quite figure out how I&#8217;ve done that, but I can&#8217;t really complain, either. It&#8217;s a good thing. Except that damned 8th Street hill. It&#8217;s just evil. It&#8217;s not as bad as other hills in town I have to say, but it still sucks the life out of you.<br />
Following a shower, Alex packed me in her car, and off we went. Destination unknown. Quite literally. This weekend was my chance to be surprised. I don&#8217;t ask questions, though &#8212; I prefer to try and guess to myself. The process is usually pretty simple elimination. I can eliminate quite a large number of things without much effort. Alex, of course, knows this. So she makes sure to keep me guessing.<br />
Our first stop was Banff. Dinner. Nothing serious. Kind of an eat-and-run thing. The Wolf Street Grill completely ignored us, so we tried some Japanese place a couple of blocks away. (The sign out front, in Japanese, solicited a great deal of laughter from Japanese tourists. I&#8217;m now dying to know what it says.) We ate quickly before heading back to the car and our exit.<br />
Although we ended up making one side-trip into Abomindable Sports. Sandal-searching. And I found something &#8212; Tevas. My previous pair of leather sandals was Teva, and I rather like them. I wanted something that wasn&#8217;t leather, though, since I wanted to be able to use these in wet conditions (among other things). Tevas ain&#8217;t cheap, though. But they fit, and they don&#8217;t hurt. That&#8217;s a tough thing to overcome sometimes. Tevas it was&#8230;<br />
We continued past Lake Louise and the Icefields Parkway turnoff (much to my relief), then over the Big Hill into Field, BC. Passing Field, we headed into the valley towards Golden. I was trying to figure out where we were going, and constantly drawing a blank. There was a faint hope that it had something to do with 2816, since I knew she was in the neck of the woods during the weekend. But beyond that, I hadn&#8217;t a clue.<br />
It seems that finally someone had the sense to start work on the Trans Canada highway east of Golden. It&#8217;s been a mess there for decades, and is generally difficult to go through. The hills are steep, the road windy, and there are a number of 40 km/h corners. The new section that they&#8217;re building, so it looks, will create a massive fly-over to reduce the cornering and hopefully twin the lanes. It&#8217;ll be a few years until it&#8217;s open though &#8212; there&#8217;s a lot of work to do.<br />
Being the master of deception that she&#8217;s becoming, Alex really threw me off with a visit to the 7-11 in Golden before going a mere two blocks to the Travelodge for our motel room for the night. I had no idea what was going on. Thoroughly stumped.<br />
The next morning we were back on the road, following a brief stop at the Humpty&#8217;s for breakfast, we continued west. Why we were going to Revelstoke, I had no idea. We stopped briefly at the railway museum (but didn&#8217;t go in &#8212; next time, I need to actually go in there). I had heard about a trip from Revelstoke to Golden with 2816, and thought that maybe that&#8217;s what we&#8217;d gone there. (Alex had mentioned that we needed to be &#8220;there&#8221; for 13:00.) We hopped back in the car, though, and continued south.<br />
Now I was stumped. I have no idea what&#8217;s south of Revelstoke. As it turned out, not much &#8212; we were on the wrong road. We ended up having to double-back to the Trans Canada and continue west. I was beside myself with confusion. Where the hell were we going?<br />
Three Valley Gap. That&#8217;s when the surprise revealed itself. As we took seats in the restaurant there, there were settings for four, not two. Someone was going to meet us &#8212; but who? I couldn&#8217;t think of anyone who lived in the area. And that&#8217;s when Allen and Jean appeared. The sneaky trio (apparently mostly Jean&#8217;s doing) had completely orchestrated the charade.<br />
The four of us had a lovely (long) lunch, followed by a transfer of inheritances that Allen and Jean had brought from Ruskin. We&#8217;ve now got some of Christena&#8217;s things that she&#8217;d left to Alex. Oddly enough, with no planning at all, they seem to fit with our house.<br />
Naps were next. Well, for three of the four of us. Not yet being tired, I went wandering with my camera. I was lucky enough to catch a west-bound coal train (the Canadian Pacific line runs right past the hotel), though I botched the timing and the train&#8217;s a bit blurry. Things to remember for the next time I try that. Fortunately, the other trains I shot don&#8217;t move.<br />
Three Valley Gap is the oddest place you&#8217;ll ever see. It&#8217;s very much a work in progress and comes complete with a variety of antiques, including five former passenger cars from the Canadian National and Canadian Pacific railroads (and technically VIA, but they&#8217;re ex-CN/CP). Included is the Eatonia, Sir William Eaton&#8217;s private railway car. Most are in poor condition, not being maintained at all and restoration seems not to be a consideration. They do make for interesting photography, though.<br />
I found the group in the main garden wandering about. We gathered, piled into Jean&#8217;s car, and drove out to Revelstoke for dinner. Our first stop sounded good on paper. But the rather angry dog and very unassuming appearance of the restaurant suggested we go elsewhere. This led us up a long windy road to the Glacier Lodge, about a stone&#8217;s throw from the Revelstoke Dam on the west side. A massive log hotel built in 1999, and apparently a great place for dinner.<br />
It was an outstanding dinner. Miso-marinated salmon encrusted with sesame and ginger. It was a powerful but tasty meal. Alex had a lamb shank done Moroccan-style, but became more intrigued with the joint she found in the meat than actually eating it. This tortured poor Boulder, one of the wilderness guide&#8217;s two dogs. He wasn&#8217;t allowed on the patio, but the proudly displayed bone was too much for him to handle. When the waitress scolded Boulder, I suggested that she scold my wife instead for being a tease.<br />
Night photography was marginally successful. The biggest success was capturing an east-bound sulphur train, the lights creating glowing lines across the frame and glinting across the water.<br />
I was woken with a &#8220;Happy Birthday&#8221;. The goal was to try and chase 2816 from Palliser to Calgary. Alex would be on her first train chase. But first, we had to have breakfast. We met up with Allen and Jean &#8230; and a few presents.<br />
Sometime in the next while, the group of us will go to Washington and ride the dinner train. That was my present from Allen and Jean. Very nice of them. Alex surprised me with a painting of the York Railway station, circa 1930s/1940s. The kicker is that I sort of knew that she&#8217;d bought <em>something</em> when we were there, but had been told I couldn&#8217;t look at it. Never even thought about it after we got back from England and had since forgotten about it. It was a very welcome surprise.<br />
The Empress had left Revelstoke heading east around 8:00. We passed her arriving at the MacDonald tunnel entrance heading into Roger&#8217;s Pass. In theory, we could have started the chase there. But I wanted to get to Palliser before her to set up, and I knew that the construction traffic would slow us down a bit.<br />
We stopped at Tim Horton&#8217;s for a light snack and teas. As expected, the construction did in fact slow us down. I kept a weather ear out for the train, trying to make sure it wouldn&#8217;t get past us. We arrived at Palliser and took up a position and waited. And waited. And waited.<br />
2816 was supposed to leave Golden at 13:00. At first, I believed that she was just behind schedule. Freight trains get priority in most cases, so it was concievable that a delay would put her off. But 90 minutes somehow didn&#8217;t seem right. Before long, I came to believe that, somehow, she&#8217;d gotten in front of us. Perhaps in Golden, when we were at the Tim&#8217;s. Heartbroken at having to give up on what should have been a great chase, we headed east without train pictures in hand.<br />
As it turned out, we were ahead of 2816. The delay was long, and had we stayed for another 10 minutes, we&#8217;d have gotten the picture. My friend Graham was there not long after we&#8217;d left. I missed a train. Me. The Trainstalker. I feel so ashamed. It put a bit of a downer on my birthday. It seems really trivial, I know, but I just hate missing trains! It was my own fault, though, a lack of patience and too much indecision did me in. Otherwise, it was a good 34th birthday.<br />
Another year older. Hopefully a bit wiser. And perhaps a lot richer. Last year, I wasn&#8217;t married. Next year &#8230; who knows?<br />
[flickr]tag:threevalleygap+birthday+2006@Geoff S.[/flickr]</p>
]]></content:encoded>
					
					<wfw:commentRss>https://geoff.sowrey.org/2006/07/surprise-birthday-weekend-in-revelstoke/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
		<post-id xmlns="com-wordpress:feed-additions:1">441</post-id>	</item>
		<item>
		<title>BC Vacation: Fairmont Hot Springs to Ruskin</title>
		<link>https://geoff.sowrey.org/2004/09/bc-vacation-fairmont-hot-springs-ruskin/?utm_source=rss&#038;utm_medium=rss&#038;utm_campaign=bc-vacation-fairmont-hot-springs-ruskin</link>
					<comments>https://geoff.sowrey.org/2004/09/bc-vacation-fairmont-hot-springs-ruskin/#respond</comments>
		
		<dc:creator><![CDATA[Geoff]]></dc:creator>
		<pubDate>Sun, 05 Sep 2004 04:28:44 +0000</pubDate>
				<category><![CDATA[General]]></category>
		<category><![CDATA[Travel]]></category>
		<category><![CDATA[british columbia]]></category>
		<category><![CDATA[fairmont hot springs]]></category>
		<category><![CDATA[golden]]></category>
		<category><![CDATA[ruskin]]></category>
		<guid isPermaLink="false">http://www.sowrey.org/index.php/2004/09/04/bc-vacation-fairmont-hot-springs-ruskin/</guid>

					<description><![CDATA[Cory had called me yesterday. I didn&#8217;t know until this morning because I hadn&#8217;t checked my cell phone. I&#8217;d had a dream that things had gone horribly awry at the office, and for some reason, that had made me want to check my cell phone. Good or bad, something was up. Sure enough, our newest &#8230; <a href="https://geoff.sowrey.org/2004/09/bc-vacation-fairmont-hot-springs-ruskin/" class="more-link">Continue reading<span class="screen-reader-text"> "BC Vacation: Fairmont Hot Springs to Ruskin"</span></a>]]></description>
										<content:encoded><![CDATA[<p>Cory had called me yesterday. I didn&#8217;t know until this morning because I hadn&#8217;t checked my cell phone. I&#8217;d had a dream that things had gone horribly awry at the office, and for some reason, that had made me want to check my cell phone. Good or bad, something was up. Sure enough, our newest star hire had backed out &#8212; an extremely disappointing turn of events. I wouldn&#8217;t talk to Cory about it until late in the afternoon.<br />
We were gone before 8:00. (We were to have left at 6:00, then 7:00, but last night&#8217;s arrival in Fairmont pushed us even later.) Allen was in Jean&#8217;s Honda Civic, Alex and I in Alex&#8217;s car. I drove while Alex read &#8220;Why I Hate Canadians&#8221;. Today&#8217;s lesson? Canada&#8217;s cast system, more formally known as how poorly we treat Indians. It&#8217;s a hard thing for a Canadian to admit that we don&#8217;t treat everyone fairly and equally. For a country known as being &#8220;nice&#8221;, we treat our aboriginal peoples very poorly. We&#8217;ve marginalized, forcibly contained, abducted and tried to convert them, even poisoned them with disease. And yet after 400 years, they&#8217;re still who they are.<br />
<span id="more-382"></span>A lot of people think Indians have it easy here in Canada &#8212; pay no taxes, live on welfare, do what they please when they want. Lest we forget that we made them wards of the state &#8212; they *have* to live this way, because there is no other way for them to live. We never gave them a chance.<br />
(Incidentally, this is a view I&#8217;ve long held, and the book has since backed up. Ferguson is definitely a socialist who supports equal rights, nationalism (he rather dislikes the ill-guided attempts of Quebec separatists), and believes the monarchy should take a hike.)<br />
When we arrived in Golden, right behind Allen&#8217;s car, we headed to the Humpty&#8217;s on the Trans-Canada Highway for breakfast. I&#8217;d never been to Humpty&#8217;s before, having always viewed it as a lower-end sort of place to obtain a meal. My opinion of that hasn&#8217;t changed (the state of the bacon alone held me to that perspective).<br />
The weather yo-yo&#8217;d constantly from the time we left Fairmont until we arrived in the Lower Mainland. It was cloudy, rainy, spittling, sunny, warm, foggy, and windy all at various times (and varying combinations) throughout the trip. Alex seemed to get the bulk of the bad weather from Golden to Revelstoke.<br />
I read &#8220;Mythic Journeys&#8221; aloud. I disagreed with some of the interpretation, and wasn&#8217;t too thrilled with the depth &#8212; quite often, there was more commentary than story. It doesn&#8217;t make for great reading.<br />
We stopped in Revelstoke for drinks and snacks, and once again changed drivers. We stopped barely five kilometres past Revelstoke to make reservations at the Three Valley Gap hotel, which is a bit of a landmark in the area, and one of Alex&#8217;s favourite locations to be. (It&#8217;s a long story, which I&#8217;m going to forego with repeating here.) We&#8217;ll be stopping there on our way back.<br />
Off to Kamloops we went. We would have driven straight through, had we not also come across Craigellachie (&#8220;cray-GAL-ah-kee&#8221;), which was where the last spike was driven on the Canadian Pacific Railway in 1886. There&#8217;s not much there other than a cairn, bearing stones from every province and territory, and a stone from Craigellachie, Scotland, where most of the original directors of the CPR hailed from.<br />
We didn&#8217;t stay long. Just enough for a couple of photos, and a chance to mock some idiotic 20-something tourists from Ontario. They were being typical of that age: stupid. Though I never remember being quite so dumb that I would have inspired someone else to think of me: &#8220;Look at me, I&#8217;m an idiot!&#8221;<br />
Pulling into Kamloops about an hour later, running almost on fumes, we headed for the ABC Country restaurant on the junction of the Trans-Canada and Highway 5A. When we went inside, I realized that it was the same restaurant where Greg and I had eaten breakfast four and a half years before, when he helped me move from Vancouver to Calgary (see [[Left Vancouver, Job, and Girlfriend for New Start in Calgary]]). This time, thankfully, I wasn&#8217;t doing anything even remotely traumatic.<br />
Alex took over for the last stint of driving to Ruskin (a hamlet next to Maple Ridge, BC), where her father lives. The trip was much shorter than I had expected &#8212; for some reason, I thought the distance from Kamloops to Hope was much longer than we experienced. Mind you, the distance from Hope to Highway 11 seemed almost as long. (It didn&#8217;t help that we got stuck in traffic.)<br />
Dairy Queen made a call to us as we pulled off the highway. We&#8217;d been dying for a Blizzard since half-way down the Coquihalla Highway, but had missed the turn-off in Hope. Fortunately, the one on Highway 11 would suffice &#8230; even if some of the patrons were a little strange.<br />
Continuing up Highway 11, across the Fraser River (and getting a dirty look from a police officer at a radar trap for going a little too quickly), we turned west on Highway 7. Another 15 minutes, and we arrived at the farm &#8212; Allen and Jean&#8217;s new home, built on the former location of Alex&#8217;s grandparent&#8217;s home.<br />
We got a tour almost as soon as we were in the house. It&#8217;s an impressive structure. It looks unassuming from the outside, but there is much hidden under the covers. First and foremost, it&#8217;s a green building. It&#8217;s made with the most energy-efficient materials available, has radiant heating in the floors, recycling wood for the flooring upstairs, low-wattage lights, and nice, large windows to allow sunlight to come in during the winter.<br />
But the best feature was found tucked away, behind the garage: geothermic heating. This is the créme de la créme of cool (or heat, depending on what season you&#8217;re in). Because the house is built on an acreage, there is sufficient space to build this. (I could probably do this in Calgary, but I shudder to think at what cost.) A space about 25 feet wide, 80 feet long, and about seven feet deep was dug out behind the house and filled with long spiral tubes, and partially filled with clay. The tubes are then filled with a vicous liquid which tranfers heat in and out of the house. The system also hooks into the hot water tank to cut down on the energy needed to keep the water hot.<br />
The end result? Theoretically, a lower energy bill. If Allen has his way, he&#8217;ll also build a windmill to generate some additional electricity, and maybe even cut his BC Hydro bill to zero.<br />
A dream worth shooting for, indeed.<br />
Alex and I ducked out for a quick tour of the yard. The acreage isn&#8217;t huge (about 6.2 acres), with the main property lot about 200 feet on both sides, and then extending about 30-50 feet into the woods and down to the creek. The landscaping potential of this place is amazing &#8212; I have great dreams of a staircase and deck space down at the creek level. Now I just need to convince Allen that it&#8217;s a good idea.<br />
I pitched the tent (Alex and I are sleeping outdoors) under four great cedar trees on the east side of the house. The ground underneath is spongy from decades of cedar leaves and soil growth. Tonight, we will sleep well.<br />
I certainly hope so, because although today&#8217;s trip was outstanding, it really wore me down.<br />
[flickr]tag:craigellachie+2004@Geoff S.[/flickr]</p>
]]></content:encoded>
					
					<wfw:commentRss>https://geoff.sowrey.org/2004/09/bc-vacation-fairmont-hot-springs-ruskin/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
		<post-id xmlns="com-wordpress:feed-additions:1">382</post-id>	</item>
	</channel>
</rss>
